mscore/file.cpp

```diff
@@ -1602,6 +1602,7 @@ void MuseScore::exportFile()
 #endif
       fl.append(tr("Standard MIDI File") + " (*.mid)");
       fl.append(tr("MusicXML File") + " (*.xml)");
+      fl.append(tr("MusicXML 3.1+ File") + " (*.musicxml)");
```

Comment:
MusicXML is already hard to convey, I would not add more complexity with a version number. If we move the .musicxml, let's use .musicxml all the way for export and call it MusicXML File.